apps/api/src/config/env-validator.ts

```diff
@@ -31,6 +31,9 @@ const validators: { [K in keyof any]: ValidatorSpec<any[K]> } = {
     default: '',
   }),
   MONGO_URL: str(),
+  MONGO_MAX_POOL_SIZE: num({
+    default: 700,
```

Comment:
Considering you are being warn in MongoDB Atlas that is over 500, maybe that should be the default limit. Same, for the .env files.
For Novu's own configuration we can set that feature flag once this is merged.